### PR TITLE
Add README for docs

### PR DESCRIPTION
**Pull request by natke**
**Description**: Add instructions for updating docs website.

Initial version. We can update with instructions for building locally and staging, as necessary.

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# How to update ONNX Runtime docs
+
+The docs for ONNX Runtime are published at https://onnxruntime.ai/docs.
```
Comment (edgchen1) [inline docs/README.md:3]: some of the docs still exist in the docs/ directory in other branches (e.g. master). what's the difference between those and "the docs" mentioned here?

Comment (faxu) [reply docs/README.md:3]: Docs in the repo should be more for technical reference, for developers of ORT source code. All customer-facing documentation content should eventually be moved to the gh-pages page.

Comment (edgchen1) [reply docs/README.md:3]: thanks for the clarification. that may be good to mention here

**Review by faxu — APPROVED** [review]

**Pull request merged**